### ISSUE TITLE
[CI] Inductor dashboards failing due to unused --quant arg

### ISSUE THREAD

**Issue by jataylo** [pytorch/pytorch#169663]
### 🐛 Describe the bug

The offending code was added in https://github.com/pytorch/pytorch/pull/123419 which results in a failure as no quant type is provided.
https://github.com/pytorch/pytorch/blob/a097e166db7077f1e8da94757ccd91a6a521550e/.ci/pytorch/test.sh#L767

This is causing unnecessary headaches when debugging inductor-dashboard runs. @huydhn is it possible for us to either provide a valid quant type or remove this?

Example failure: https://ossci-raw-job-status.s3.amazonaws.com/log/56856027123 for AMD and similar on https://ossci-raw-job-status.s3.amazonaws.com/log/56764730017 for NV.
```
2025-12-02T02:00:17.0782532Z + python benchmarks/dynamo/timm_models.py --performance --cold-start-latency --inference --quant --backend inductor --device cuda --total-partitions 7 --partition-id 2 --output /var/lib/jenkins/pytorch/test/test-reports/inductor_cudagraphs_low_precision_timm_models_quant_inference_rocm_performance.csv
2025-12-02T02:00:19.6203822Z        [--channels-last]
2025-12-02T02:00:19.6204176Z        [--batch-size BATCH_SIZE]
2025-12-02T02:00:19.6204547Z        [--iterations ITERATIONS]
2025-12-02T02:00:19.6204936Z        [--batch-size-file BATCH_SIZE_FILE]
2025-12-02T02:00:19.6205320Z        [--cosine]
2025-12-02T02:00:19.6205614Z        [--freezing]
2025-12-02T02:00:19.6205953Z        [--inductor-config INDUCTOR_CONFIG]
2025-12-02T02:00:19.6206333Z        [--ci]
2025-12-02T02:00:19.6206618Z        [--dashboard]
2025-12-02T02:00:19.6206949Z        [--skip-fp64-check]
2025-12-02T02:00:19.6207276Z        [--fast]
2025-12-02T02:00:19.6207565Z        [--only ONLY]
2025-12-02T02:00:19.6207883Z        [--multiprocess]
2025-12-02T02:00:19.6208197Z        [--ddp]
2025-12-02T02:00:19.6208490Z        [--fsdp]
2025-12-02T02:00:19.6208833Z        [--optimize-ddp-mode OPTIMIZE_DDP_MODE]
2025-12-02T02:00:19.6209530Z        [--distributed-master-port DISTRIBUTED_MASTER_PORT]
2025-12-02T02:00:19.6209993Z        [--dynamic-shapes]
2025-12-02T02:00:19.6210361Z        [--propagate-real-tensors]
2025-12-02T02:00:19.6210749Z        [--dynamic-batch-only]
2025-12-02T02:00:19.6211114Z        [--specialize-int]
2025-12-02T02:00:19.6211449Z        [--use-eval-mode]
2025-12-02T02:00:19.6211801Z        [--skip-accuracy-check]
2025-12-02T02:00:19.6212189Z        [--generate-aot-autograd-stats]
2025-12-02T02:00:19.6212593Z        [--inductor-settings]
2025-12-02T02:00:19.6213079Z        [--suppress-errors]
2025-12-02T02:00:19.6213417Z        [--output OUTPUT]
2025-12-02T02:00:19.6213816Z        [--output-directory OUTPUT_DIRECTORY]
2025-12-02T02:00:19.6214221Z        [--disable-output]
2025-12-02T02:00:19.6214560Z        [--baseline BASELINE]
2025-12-02T02:00:19.6214912Z        [--part PART]
2025-12-02T02:00:19.6215259Z        [--export-profiler-trace]
2025-12-02T02:00:19.6215725Z        [--profiler-trace-name PROFILER_TRACE_NAME]
2025-12-02T02:00:19.6216164Z        [--profile-details]
2025-12-02T02:00:19.6216514Z        [--export-perfdoctor]
2025-12-02T02:00:19.6216885Z        [--diff-branch DIFF_BRANCH]
2025-12-02T02:00:19.6217240Z        [--tag TAG]
2025-12-02T02:00:19.6217536Z        [--explain]
2025-12-02T02:00:19.6217826Z        [--stats]
2025-12-02T02:00:19.6218144Z        [--use-warm-peak-memory]
2025-12-02T02:00:19.6218510Z        [--print-memory]
2025-12-02T02:00:19.6218865Z        [--print-compilation-time]
2025-12-02T02:00:19.6219263Z        [--print-dataframe-summary]
2025-12-02T02:00:19.6219651Z        [--disable-cudagraphs]
2025-12-02T02:00:19.6220033Z        [--disable-split-reductions]
2025-12-02T02:00:19.6220450Z        [--disable-persistent-reductions]
2025-12-02T02:00:19.6220874Z        [--disable-divisible-by-16]
2025-12-02T02:00:19.6221324Z        [--inductor-compile-mode INDUCTOR_COMPILE_MODE]
2025-12-02T02:00:19.6221782Z        [--print-graph-breaks]
2025-12-02T02:00:19.6222146Z        [--log-graph-breaks]
2025-12-02T02:00:19.6222495Z        [--trace-on-xla]
2025-12-02T02:00:19.6222842Z        [--xla-tolerance XLA_TOLERANCE]
2025-12-02T02:00:19.6223230Z        [--collect-outputs]
2025-12-02T02:00:19.6223614Z        [--enable-activation-checkpointing]
2025-12-02T02:00:19.6224005Z        [--timing]
2025-12-02T02:00:19.6224298Z        [--progress]
2025-12-02T02:00:19.6224607Z        [--timeout TIMEOUT]
2025-12-02T02:00:19.6225046Z        [--per_process_memory_fraction PER_PROCESS_MEMORY_FRACTION]
2025-12-02T02:00:19.6225545Z        [--no-translation-validation]
2025-12-02T02:00:19.6225913Z        [--minify]
2025-12-02T02:00:19.6226225Z        [--compiled-autograd]
2025-12-02T02:00:19.6226595Z        [--profile_dynamo_cache_lookup]
2025-12-02T02:00:19.6226979Z        [--snapshot-memory]
2025-12-02T02:00:19.6227313Z        [--retain-output]
2025-12-02T02:00:19.6227656Z        [--caching-precompile]
2025-12-02T02:00:19.6228230Z        [--save-model-outputs-to SAVE_MODEL_OUTPUTS_TO]
2025-12-02T02:00:19.6228782Z        [--compare-model-outputs-with COMPARE_MODEL_OUTPUTS_WITH]
2025-12-02T02:00:19.6229340Z        [--cold-start-latency | --warm-start-latency]
2025-12-02T02:00:19.6229761Z        [--nnc]
2025-12-02T02:00:19.6230104Z        [--float16 | --bfloat16 | --float32 | --amp]
2025-12-02T02:00:19.6230550Z        [--amp-dtype {bfloat16,float16}]
2025-12-02T02:00:19.6230952Z        [--verbose | --quiet]
2025-12-02T02:00:19.6235126Z        [--coverage | --overhead | --speedup-dynamo-ts | --speedup-fx2trt | --speedup-fx2trt-fp16 | --print-fx | --print-aten-ops | --inductor | --optimus {vertical_opt,horizontal_opt,all} | --quantization {int8dynamic,int8weightonly,int4weightonly,autoquant,noquant} | --export | --export-aot-inductor | --aot-precompile | --export-nativert | --torchscript-jit-trace | --xla | --backend {aot_eager,aot_eager_decomp_partition,aot_eager_decomp_partition_crossref,aot_eager_decomp_partition_with_mode,aot_eager_default_partitioner,aot_ts,cudagraphs,dynamo_accuracy_minifier_backend,dynamo_minifier_backend,eager,eager_debug,eager_noexcept,inductor,non_leaf_compile_error_TESTING_ONLY,openxla,openxla_eval,pre_dispatch_eager,relu_accuracy_error_TESTING_ONLY,relu_compile_error_TESTING_ONLY,relu_runtime_error_TESTING_ONLY,ts,tvm} | --nothing | --log-conv-args | --recompile-profiler | --find-batch-sizes]
2025-12-02T02:00:19.6239390Z        (--accuracy | --performance | --tolerance)
2025-12-02T02:00:19.6239822Z        (--training | --inference)
2025-12-02T02:00:19.6240340Z timm_models.py: error: argument --quantization: expected one argument
``

### Versions

Upstream CI
